src/GitcoinGovernor.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+// forgefmt: disable-start
+import { GovernorCountingSimple, Governor } from "openzeppelin-contracts/governance/extensions/GovernorCountingSimple.sol";
+import { GovernorVotesComp, ERC20VotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
```

Comment:
just sorting alphabetically
```suggestion
import { ERC20VotesComp, GovernorVotesComp } from "openzeppelin-contracts/governance/extensions/GovernorVotesComp.sol";
```

Reply:
done in 78a11b444add9c6e0fb59e6751995cb55d6c82dd